### PR TITLE
Added l2vpn network rule

### PR DESCRIPTION
**Pull request by gvarao**

### CODE REVIEW

**Comment by gvarao** [comment]
jenkins please retry a build

File: juniper_official/vpn/evpn/evpn-view-nw.rule [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+ * and notifies anomalies when either evpn session or PE interface is down.
+ *
+ * Requirement 455 - https://paragon-automation.atlassian.net/browse/REQ-455
+ *
+ */
+healthbot {
+    topic evpn {
+        description "EVPN network rule to monitor evpn neighbors and notify anomalies";
+        synopsis "EVPN state analyzer";
+        rule evpn-view {
```
Comment (gaddevijay) [inline juniper_official/vpn/evpn/evpn-view-nw.rule]: shouldn't this be check-even-view?

Comment (gvarao) [reply juniper_official/vpn/evpn/evpn-view-nw.rule]: Modified

File: juniper_official/vpn/evpn/evpn-view-nw.rule [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                    then {
+                        status {
+                            color green;
+                            message "$pe-router-name, VPN: $vpn-name, Neighbor-state: up";
+                        }
+                    }
+                }
+                term is-vpn-session-up {
+                    when {
+                        less-than "$vpn-state" 0;
```
Comment (gaddevijay) [inline juniper_official/vpn/evpn/evpn-view-nw.rule]: What about equal to 0?

Comment (gvarao) [reply juniper_official/vpn/evpn/evpn-view-nw.rule]: Changed

**Review by gaddevijay — APPROVED** [review]

**Pull request merged**